gradio/components.py

```diff
@@ -2686,6 +2286,8 @@ def postprocess(
             JSON object with key 'headers' for list of header names, 'data' for 2D array of string or numeric data
         """
         if y is None:
+            return self.postprocess(self.test_input)
```

Comment:
Not totally sure if this makes sense, but this is kinda what we were doing before